html/semantics/forms/the-fieldset-element/accessibility/legend-visibility-hidden-manual.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<title>fieldset accessibility test: legend visibility: hidden</title>
+<style>
+ legend { visibility: hidden; }
```

Comment:
might be worth to have display:none/visibility:hidden placed on fieldset element. Also might worth testing visibility:collapsed and visibility:visible